framework/src/test/java/org/tron/core/db/ManagerTest.java

```diff
@@ -928,14 +952,22 @@ public void switchBack()
     Args.setParam(new String[]{"--witness"}, Constant.TEST_CONF);
     long size = chainManager.getBlockStore().size();
     System.out.print("block store size:" + size + "\n");
-    String key = "f31db24bfbd1a2ef19beddca0a0fa37632eded9ac666a05d3bd925f01dde1f62";
+    String key = PublicMethod.getRandomPrivateKey();;
```

Comment:
`;;`, need to remove.

Reply:
@halibobo1205 
This is obviously bad code, but it must be changed in the release branch or new conflicts will be created.